inc/customizer.php

```diff
@@ -363,6 +363,40 @@ function hale_customize_register( $wp_customize ) {
 		)
 	);
 
+	$wp_customize->add_setting(
+		'heading_colour',
+		array(
+			'default'           => '#0b0c0c',
+			'sanitize_callback' => 'sanitize_hex_color',
+		)
+	);
+	$wp_customize->add_control(
+		'heading_colour',
+		array(
+			'label'       => esc_html__( 'Heading colour', 'hale' ),
+			'description' => esc_html__( 'Choose a colour for H1 and H2 titles', 'hale' ),
+			'section'     => 'colors',
+			'type'        => 'text',
+		)
+	);
+
+	$wp_customize->add_setting(
+		'cookie_button_colour',
+		array(
+			'default'           => '#00703c',
+			'sanitize_callback' => 'hale_sanitize_nohtml',
```

Comment:
This sanitize callback will mean colour names (e.g. "red") are also allowed, but if it doesn't match a colour then it will cause errors.  I think this is a more risky approach so I don't think we should do this one.